docs/source/io/introduction.rst

```diff
@@ -0,0 +1,27 @@
+io
```

Comment:
Sorry I did not try to build the documentation earlier.
The all-small-cap and short section title looks slightly out of place:
![Screenshot 2020-11-25 at 14 39 55](https://user-images.githubusercontent.com/3673984/100241898-1b6b1080-2f2c-11eb-8e35-4e3d18539874.png)

Would "File I/O" look a bit better? And "io.h5" be changed to "HDF5 File support"?